<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>835</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(233, 245, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="labelListaTitulo">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>30</y>
      <width>231</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>LISTAGEM DE PEDIDOS</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>430</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="labelPeriodoInicio">
       <property name="text">
        <string>Período de início:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="linePeriodoInicio">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>430</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="labelPeriodoFinal">
       <property name="text">
        <string>Período final: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="linePeriodoFinal">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="btnPesquisa">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>430</y>
      <width>131</width>
      <height>28</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>120</width>
      <height>0</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#ffa42f</string>
    </property>
    <property name="text">
     <string>PESQUISAR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnGeraCSV">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>510</y>
      <width>131</width>
      <height>28</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>120</width>
      <height>0</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #22ee88</string>
    </property>
    <property name="text">
     <string>GERAR PLANILHA</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tablePedidos">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>81</y>
      <width>631</width>
      <height>321</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <column>
     <property name="text">
      <string>id</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>codigo</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>nome</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>descricao</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Total</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="btnGeraPDF">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>470</y>
      <width>131</width>
      <height>28</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>120</width>
      <height>0</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#ffccee</string>
    </property>
    <property name="text">
     <string>GERAR PDF</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>835</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
